src/common/powers_array.cairo

```diff
@@ -0,0 +1,19 @@
+// Writes the following `n` values into array:
+//   cur, cur * alpha, cur * alpha^2, ..., cur * alpha^(n - 1).
+fn powers_array(cur: felt252, alpha: felt252, n: u32) -> Array<felt252> {
```

Comment:
I would rename `cur` to `initial` or `first`

Reply:
let's go with `initial`, seems nice